crates/red_knot_python_semantic/resources/mdtest/narrow/issubclass.md

```diff
@@ -112,8 +112,7 @@ def _(flag: bool):
         reveal_type(t)  # revealed: Literal[NoneType]
 
     if issubclass(t, type(None)):
-        # TODO: this should be just `Literal[NoneType]`
-        reveal_type(t)  # revealed: Literal[int, NoneType]
+        reveal_type(t)  # revealed: Literal[NoneType]
```

Comment:
I love these unexpected TODO fixes, where multiple features compose together and it just... does the right thing